File: Advance Data Structures_03-12-2024/13) Binary Tree/10) Convert A Binary Tree Into Its Mirror Tree/Convert A Binary Tree Into Its Mirror Tree.cpp
void findMirror(Node* root) {
    if (root == nullptr) {
        return;
    }
    Node* temp = root->left;
    root->left = root->right;
    root->right = temp;
    findMirror(root->left);
    findMirror(root->right);
}
void inorder(Node* root) {
    if (root == nullptr) {
        return;
    }
    inorder(root->left);
    cout << root->data << " ";
    inorder(root->right);
}

File: Advance Data Structures_03-12-2024/13) Binary Tree/10) Convert A Binary Tree Into Its Mirror Tree/Convert A Binary Tree Into Its Mirror Tree.java
static Node findMirror(Node root) {
    if (root == null) {
        return null;
    }
    Node temp = root.left;
    root.left = root.right;
    root.right = temp;
    findMirror(root.left);
    findMirror(root.right);
    return root;
}
static void inorder(Node root) {
    if (root == null) {
        return;
    }
    inorder(root.left);
    System.out.print(root.data + " ");
    inorder(root.right);
}
